contrib/lodepng-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set (SOURCE_DIR ${ClickHouse_SOURCE_DIR}/contrib/lodepng)
```

Comment:
Since we create bitmap at first, we can use any encoding format. Maybe we can consider [ppm](https://en.wikipedia.org/wiki/Netpbm#File_formats) image format, it's really simple and doesn't require any library. User can convert it to any format as he want, e.g using ImageMagic `clickhouse-client -q 'SELECT * FROM t FORMAT ImagePpm' | convert - img.png`